### PR TITLE
Bump a few thirdparty libs

### PR DESCRIPTION
**Pull request by NiLuJe**
* libpng 1.6.35
* FBInk 1.2.0

### CODE REVIEW

**Comment by NiLuJe** [comment]
ping @Frenzie : What happens re: https://github.com/NiLuJe/koreader-base/commit/e88f4539336f042aadc7ae54373b2498521881f8 when there's no libtool in the package tree, and it instead relies on the system's?

(Because that's what happens here)

Also, were you aware of this:  https://git.midipix.org/cgit.cgi/slibtool

(Have I mentioned that I despise libtool? :D)

**Comment by NiLuJe** [comment]
False alert, I'm stupid.

**Comment by NiLuJe** [comment]
Still leaving that here, for posterity's sake: https://git.midipix.org/cgit.cgi/slibtool (I despise libtool).

**Comment by Frenzie** [comment]
libtool > Android /whistles

File: thirdparty/libpng/CMakeLists.txt [hunk 1]
```diff
@@ -28,15 +28,15 @@ endif()
...
+    v1.6.35
     ${SOURCE_DIR}
 )
 
 include(ExternalProject)
 ExternalProject_Add(
     ${PROJECT_NAME}
     DOWNLOAD_COMMAND ${CMAKE_COMMAND} -P ${GIT_CLONE_SCRIPT_FILENAME}
-    PATCH_COMMAND sh autogen.sh
+    PATCH_COMMAND ""
```
Comment (Frenzie) [inline thirdparty/libpng/CMakeLists.txt]: <3 fixing things by removing stuff

Comment (NiLuJe) [reply thirdparty/libpng/CMakeLists.txt]: Yeah, my first reflex was "fuck it, let's autoreconf -fi ourselves"... and then I checked the repo at that tag, and people rejoiced! :D.

**Pull request merged**